explore-analyze/visualize/charts/line-charts.md

```diff
@@ -0,0 +1,233 @@
+   - Add a [reference line](../lens.md#add-reference-lines) to mark targets or SLOs.
+:::::
+
+:::::{step} Customize the chart to follow best practices
+You can tweak the appearance of your chart by adjusting axes, legends, and series styles from the chart settings. Consider the following best practices:
+
+**Use color wisely**
+:   Assign colors that match your users' expectations and consider your specific context.
+
+**Provide context**
+:   Add descriptive axis titles and units to explain what the chart shows.
+
+For layout, hierarchy, and color guidance on dashboards, check the EUI’s [Dashboard good practices](https://eui.elastic.co/docs/dataviz/dashboard-good-practices/). 
+For more chart configuration options, go to the [Line chart settings](#settings) section.
+:::::
+
+:::::{step} Save the chart
+- If you accessed Lens from a dashboard, select **Save and return** to save the visualization and add it to that dashboard, or select **Save to library** to add the visualization to the Visualize library.
+- If you accessed Lens from the Visualize library, select **Save**. The Save menu lets you also add the visualization to a dashboard and to the Visualize library.
+:::::
+
+::::::
+
+## Advanced line scenarios
+
+### Compare with a previous period [line-previous-period]
+Compare the current value with a prior time range using time shift to quickly see deltas.
+
+1. Create a line series for the current value, for example: `average(response_time)`.
+2. Add a second series with a time shift, for example: `average(response_time, shift='1w')`.
+3. Use the legend labels to clarify “Current” versus “Previous (1w)”.
+
+| Single series | With previous period |
+|---|---|
+| ![Metric over time](../../images/kibana-lens_lineChartMetricOverTime_8.4.0.png "title =70%") | ![Multiple series with previous period](../../images/kibana-lens_lineChartMultipleDataSeries_7.16.png "title =70%") |
+
+::::{tip}
+You can also compute the relative change as a separate series using a formula, for example:  
+`(average(response_time) - average(response_time, shift='1w')) / average(response_time, shift='1w')`
+:::: 
+
+### Smooth noisy series with moving average [line-moving-average]
+Noisy metrics (for example, per-minute throughput) can be smoothed:
+
+1. Add a series such as `sum(bytes)`.
+2. Add a second series using `moving_average(sum(bytes), window=5)`.
+3. Keep both series visible (raw versus smoothed) or show only the smoothed one.
+
+### Highlight thresholds with reference lines [line-reference-lines]
+Use reference lines to indicate SLOs or alert thresholds.
+
+1. In the chart settings, add a **Reference line** (for example, `200` ms or `0.95`).
+2. Give it a label (for example, `Target` or `SLO`), choose a color, and optionally a band.
+
+![Reference line](../../images/kibana-lens_referenceLine_7.16.png "title =70%")
```

Comment:
replace with better image

Reply:
Maria is helping to rework this scenario with meaningful data.